.github/workflows/compare.yaml

```diff
@@ -30,13 +30,43 @@ jobs:
     steps:
       - name: Git Checkout
         uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.WORKFLOW_GIT_ACCESS_TOKEN }}
       - name: Install coursier
         uses: coursier/setup-action@v1
         with:
           apps: scala-cli
 
+      - run: echo "filename=$(./scripts/get_filename.sh)" >> "$GITHUB_ENV"
+        id: filename
       - name: Generate raport
         env:
           ES_USER: ${{ secrets.OPENCB_ELASTIC_USER }}
           ES_PASSWORD: ${{ secrets.OPENCB_ELASTIC_PSWD }}
-        run: ./scripts/raport-regressions.scala ${{ inputs.scala-version }} --compareWith=${{ inputs.compare-with-scala-version }} --buildId=${{ inputs.build-id }} --compareWithBuildId=${{ inputs.compare-with-build-id }}
+        run: |
+          ./scripts/create_post_with_front_matter.sh "$HOME/${{ env.filename }}"
+          ./scripts/raport-regressions.scala ${{ inputs.scala-version }} \
+            --compareWith=${{ inputs.compare-with-scala-version }} \
+            --buildId=${{ inputs.build-id }} \
+            --compareWithBuildId=${{ inputs.compare-with-build-id }} \
+            >> "$HOME/${{ env.filename }}"
+      - name: Push raport
+        run: |
+          if [ -s "$HOME/${{ env.filename }}" ]; then
+            echo "The raport file is non-empty."
+            git checkout gh-page
+          
+            mkdir -p "./_posts"
+            cat "$HOME/${{ env.filename }}" > "./_posts/${{ env.filename }}"
+          
+            git config --global user.name "GitHub Actions"
+            git config --global user.email "mgajek@virtuslab.com"
+            
+            git add "./_posts/${{ env.filename }}"
+            git commit -m "Upload build raport"
+            git push origin gh-page
+          else
+            echo "The raport file is empty."
+            exit(0)
+          fi
```

Comment:
This workflow was added for manual, ad-hoc comparison of two arbitrary versions/builds. I think it should be better to combine it with workflow for scheduled builds: https://github.com/VirtusLab/community-build3/blame/7d56005233c7802baa8952c14cbb0771eda6b7ef/.github/workflows/buildExecuteScheduledWeekly.yaml#L68-L84 
Or even better make it a reusable action, so we can use if for scheduled builds, and manually for releases, which in the future might be present in the subpage.